templates/examples/chapar/meta.yml

```diff
@@ -0,0 +1,331 @@
+---
+fullname: Chapar
+shortname: chapar
+organization: coq-community
+community: true
+
+synopsis: >-
+  A framework for verification of causal consistency for distributed key-value stores
+  and their clients in Coq
+
+description: |
+  A framework for modular verification of causal consistency for replicated key-value
+  store implementations and their client programs in Coq. Includes proofs of the causal consistency
+  of two key-value store implementations and a simple automatic model checker for the correctness
+  of client programs.
+
+paper:
+  doi: 10.1145/2837614.2837622  
+  url: http://adam.chlipala.net/papers/ChaparPOPL16/
+  title: 'Chapar: Certified Causally Consistent Distributed Key-value Stores'
+
+authors:
+- name: Mohsen Lesani
+  initial: true
+- name: Christian J. Bell
+  initial: true
+- name: Adam Chlipala
+  initial: true
+
+maintainers:
+- name: Karl Palmskog
+  nickname: palmskog
+
+opam-file-maintainer: palmskog@gmail.com
+
+license:
+  fullname: MIT License
+  identifier: MIT
+
+plugin: false
+
+supported_coq_versions:
+  text: 8.9 or later
+  opam: '{(>= "8.9" & < "8.10~") | (= "dev")}'
+
+tested_coq_versions:
+- version_or_url: https://github.com/coq/coq-on-cachix/tarball/master
+- version_or_url: 8.9
+
+tested_coq_opam_version: dev
+
+namespace: Chapar
+
+extracted:
+  extracted_fullname: Chapar Stores
+  extracted_shortname: chapar-stores
+
+  extracted_synopsis: Three executable causally consistent distributed key-value stores
+  extracted_description: |
+    Three key-value stores, verified to be causally consistent in
+    the Coq proof assistant and extracted to executable code.
+
+  target: stores
+
+  supported_ocaml_versions:
+    text: 4.05.0 or later
+    opam: '{>= "4.05.0"}'
+
+  extracted_dependencies:
+  - opam:
+      name: ocamlbuild
+      version: '{build}'
+    nix: ocamlbuild
+    description: >-
+      [OCamlbuild](https://github.com/ocaml/ocamlbuild)
+  - opam:
+      name: batteries
+      version: '{>= "2.8.0"}'
+    nix: batteries
```

Comment:
Since you are providing Nix names for dependencies as well, I think we should try to see how does the extracted package build with Nix. But that can be done as a later step.